internal/pkg/logger/logger.go

```diff
@@ -126,7 +128,7 @@ func configure(cfg *config.Config) (zerolog.Logger, WriterSync, error) {
 	if cfg.Logging.ToStderr {
 		out := io.Writer(os.Stderr)
 		if cfg.Logging.Pretty {
-			out = zerolog.ConsoleWriter{Out: os.Stderr}
+			out = zerolog.ConsoleWriter{Out: os.Stderr, TimeFormat: "15:04:05.000"}
```

Comment:
Isn't this also used inside a docker container potentially? Should we also include the full timestamp?